test/test_pyfftw_numpy_interface.py

```diff
@@ -229,11 +229,12 @@ def _validate(self, array_type, test_shape, dtype,
 
         np_input_array = numpy.asarray(input_array)
 
-        if np_input_array.dtype == 'clongdouble':
-            np_input_array = numpy.complex128(input_array)
+        # TODO why are long double inputs copied to double precision?
```

Comment:
I can't remember the precise reason for this. There are all sorts of annoyances around `clongdouble`, particularly on windows. I suspect this sidesteps one of those issues. It might be your work supersedes this approach.

Reply:
I mean, if you're checking what types are available, that's probable enough!

Reply:
Ok, I imagine this was supposed to work even if long double wasn't available? Weird, because `long double` is required but OK, I just remove the comments as we can now explicitly check the available types